test/util/random.cpp

```diff
@@ -34,16 +25,8 @@ namespace test_util {
 
 unsigned long produce_nondeterministic_random_seed()
```

Comment:
For various reasons (i.e. process ids are 32 bit and `time()` is still only using 31 bits) we seem to have never produced a random seed of more than 32-bits. Should we ever need to, we can change it at that point.